packages/app-store/routing-forms/components/FormInputFields.tsx

```diff
@@ -26,7 +27,10 @@ export type FormInputFieldsProps = {
 export default function FormInputFields(props: FormInputFieldsProps) {
   const { form, response, setResponse, disabledFields = [] } = props;
 
-  const formFieldsQueryBuilderConfig = getQueryBuilderConfigForFormFields(form);
+  const formFieldsQueryBuilderConfig = withRaqbSettingsAndWidgets({
```

Comment:
withRaqbSettingsAndWidgets allows us to merge settings and widgets later separately where we need to render the inputs or render raqb rules.